backend/tracim_backend/tests/functional/test_workspaces.py

```diff
@@ -1191,18 +1229,36 @@ def test_api__get_workspace_members__ok_200__admin(self):
                 'admin@admin.admin'
             )
         )
-        res = self.testapp.get('/api/v2/workspaces/{}/members'.format(
+        print(str(user_id) + '##' + str(workspace_id))
```

Comment:
Print is probably a missed debug line

Reply:
you're right. For lack of unit test see https://github.com/tracim/tracim/issues/709